src/editing.md

```diff
@@ -500,17 +500,21 @@ have their own layouts specific to that language.
 European languages that use accents may have their own layout, but can
 often be typed on a generic "international keyboard" layout. These work
 by typing the accent, then the character you want accented - e.g. an
-apostrophe (') then the letter a (a) gives á.
+apostrophe (´) then the letter a (a) gives á.
 
 To add the international keyboard on Windows machines, please see
 <https://thegeekpage.com/how-to-add-us-international-keyboard-in-windows-10/>
 
 To add it on Macs, please see
 <http://www.macworld.com/article/1147039/os-x/accentinput.html>
 
+To add it on Linux, please see
+<https://wiki.archlinux.org/title/Input_method> or <https://wiki.debian.org/Keyboard#Modern_strategy>.
```

Comment:
Is this not explaining an IME which is different from what the other links are doing here?

We can organise the info better in this topic.

Reply:
Looks much better than before.